naga/src/back/spv/block.rs

```diff
@@ -200,10 +200,7 @@ impl<'w> BlockContext<'w> {
     fn is_intermediate(&self, expr_handle: Handle<crate::Expression>) -> bool {
         match self.ir_function.expressions[expr_handle] {
             crate::Expression::GlobalVariable(handle) => {
-                match self.ir_module.global_variables[handle].space {
-                    crate::AddressSpace::Handle => false,
-                    _ => true,
-                }
+                self.ir_module.global_variables[handle].space != crate::AddressSpace::Handle
```

Comment:
thought: This is also a good place to use `matches!(…)`, though I think the `PartialEq` implementation isn't going away anytime soon (so there's no reason we _have_ to use it here).

Reply:
I did think about that. But when I see `matches!` instead of `!=`, I expect pattern-matching behavior to be relevant. In this case, it's not, so it seemed like `!=` was less distracting.